### PR TITLE
Check for custom resolvers on start up

### PR DESCRIPTION
**Pull request by Liam-Doodson**
# Description

Added checks for custom resolvers for fields with the `@customResolver` directive.

# Checklist

The following requirements should have been met (depending on the changes in the branch):

- [x] Documentation has been updated
- [x] TCK tests have been updated
- [x] Integration tests have been updated
- [x] Example applications have been updated
- [x] New files have copyright header
- [x] CLA (https://neo4j.com/developer/cla/) has been signed

### CODE REVIEW

**Comment by neo4j-team-graphql** [comment]
Looks like you've updated the documentation!

Check out your changes at https://neo4j-graphql-docs-2232.surge.sh

File: packages/graphql/src/schema/get-custom-resolver-meta.test.ts [hunk 1]
```diff
@@ -43,14 +66,17 @@ describe("getCustomResolverMeta", () => {
...
         };
 
-        const result = getCustomResolverMeta(field);
+        const result = getCustomResolverMeta(field, object, resolvers);
 
         expect(result).toBeUndefined();
     });
-
-    test("should throw if requires not a list", () => {
+    test("should throw if requires not a list 123", () => {
```
Comment (angrykoala) [inline packages/graphql/src/schema/get-custom-resolver-meta.test.ts:79]: I don't understand this rename :thinking:

File: packages/graphql/tests/integration/directives/customResolver.int.test.ts [hunk 2]
```diff
@@ -80,14 +81,14 @@ describe("@customResolver directive", () => {
...
+            await session.run(`MATCH (n:${testType}) DETACH DELETE n`);
             await session.close();
         });
 
         test("removes a field from all but its object type, and resolves with a custom resolver", async () => {
             const source = `
-                query Users($userId: ID!) {
-                    users(where: { id: $userId }) {
+                query ${testType.name}($userId: ID!) {
+                    ${testType.plural}(where: { id: $userId }) {
```
Comment (angrykoala) [inline packages/graphql/tests/integration/directives/customResolver.int.test.ts:91]: :clap:

**Review by angrykoala — DISMISSED** [review]
LGTM

File: packages/graphql/src/schema/get-custom-resolver-meta.test.ts (hunk 1 above)
Comment (Liam-Doodson) [reply packages/graphql/src/schema/get-custom-resolver-meta.test.ts]: Ah yes, this test name was the same as the start of another so wasn't skipping. I obviously meant to come up with a better name though! Should be fixed now 😄

**Pull request merged**